meetings/2023-02-02.md

```diff
@@ -0,0 +1,210 @@
+* AC: If we can't release quickly we should try to get things quickly but if we can do it with more cadence then okay.
+* SC: ...
+* eP: can we clarify for every document who has to approve it. Team has to approve every version of every spec or major changed? Also whoever approved there is a change-log. Going from 0.2 to 0.3 should be a simpler change-log. The review of that shouldn't take too long. I propose to make changes small.
+* RG: can you clarify how much difference / changes to expect in the main protocol as with the LDN protocol. I don't think there'd be many.
+* eP: At least the filtering stuff we were discussing or receiver.. It can go into 0.2 or 0.3. The diff is small. Currently the person has to read the whole doc.
+* SC: I wonder if we can just push it in 0.2. On one hand we had very little discussion and implementation experience. On the other hand we feel really confident.
+* AC: These are good patterns/optional things that servers can implement or not. Low hanging fruit. If it seems useful, I don't see it harmful to add it.
+* SC: Anything else form the other notifications channel?
+* eP: The sender is undefined in some ways. We should do that before adding receiver. If we add, it should be in the main spec but don't we 
+* SC: I agree. I think this was about the identity of notification receiver. It depends how we look at it or what we need it for. We need *sendTo* to be controlled by some receiver. There is some assumption that receiver can be discovered by looking at who controls *sendTo*.
+* eP: Tracking issue for receiver can help. When we talked about this in one point, but hard to track this.
+
+ACTION: SC to create issue for `receiver`.
+
+* SC: LDN spec mimic the structure of notifications protocol. It makes it easier for anyone following it to see what it changes compared to the main protocol. It does not duplicate content just extends what is needed.
+* RG: Section 1.2.3 and 1.2.4 I think are duplicated from the main spec.
+* SC: Right, but it extends it. It says that Subscription Server is like in the main protocol but it expects it to ask as LDN receiver. So if you are authorized you should be able to see the subscriptions.
+* eP: I think we should be careful about the impact of extending the main protocol. For LDN, it might not necessary/requires to function., e.g., SubscriptionServer, but doesn't need to act as an LDN inbox. is it must for this channel to function? it might put more burden for server to support that.
+* SC: I think there are two different things. One is editorial and second the things that could be part of the protocol but it doesn't need to act as LDN receiver. Which not only accepts the payloads but also exposes the subscriptions. Some of these implementations are going to be solid servers anyways. If they behave as LDP it already covers what LDN requires. We can expect it to be implemented.
```

Comment:
```suggestion
* SC: Right, but it extends it. It says that Subscription Server is like in the main protocol, but it expects it to ask as LDN receiver. So if you are authorized, you should be able to see the subscriptions.
* eP: I think we should be careful about the impact of extending the main protocol. For LDN, it might not be necessary/required to function (e.g., SubscriptionServer), but doesn't need to act as an LDN inbox. Is it a must for this channel to function? It might put more burden on the server to support that.
* SC: I think there are two different things. One is editorial, and the other, things that could be part of the protocol but aren't necessary to act as an LDN receiver. Which not only accepts the payloads but also exposes the subscriptions. Some of these implementations are going to be Solid Servers anyway. If they behave as LDP, it already covers what LDN requires. We can expect it to be implemented.
```